Fix Issue #12

asad0206 opened the pull request:
*Fix issue #12 

- Add: Upload Page Redesign
- Added components: ShadCN Card component
- File Edited: page.tsx, components/ui/upload.tsx


Please check the file and the UI design.
I have checked for the responsiveness of the page. In case of any missed out bugs you can let me know.
Made the uploader page a separate component so the main "PAGE.TSX" file remains clean.
I did not install any of the backend as i was confused. Please provide help if you can

![localhost_3000_](https://github.com/user-attachments/assets/40c698a4-dc65-4fac-9769-f1297f826b24)

asad0206 commented:
Hi @aadeshkulkarni  The PR is ready for review.

aadeshkulkarni commented:
@asad0206 - This looks okay, but it needs to be visually stimulating.

Also, there are a lot of formatting changes in your PR. Can you please rollback them so that it's easier to review your changes, please ?

At the moment, it's difficult to figure out which is a formatting change and which is PR related.

Thank you.

asad0206 commented:
Can you tell me what other changes would u like in the UI? any details for what i can add.

And I will also try to fix the changes in PR. please give me some time.

asad0206 committed:
adding the shadcn card ui

asad0206 committed:
Making uploader component and adding it to page.tsx

asad0206 commented:
- I have tried to address all the other mentioned issues

Have change the background gradient

![image](https://github.com/user-attachments/assets/d7f8a76b-7861-4186-b1a0-ddcffa73cebc)


Will try to add an bg image to make it look appealing .

asad0206 committed:
adding backdrop images

asad0206 commented:
- I have tried to address all the issues
- Added 2 images for background so that it looks appealing
- Changed the Glassmorphic card from Slate theme to white theme to make it visible with darker backgrounds

an example it given below

![image](https://github.com/user-attachments/assets/10a8f550-8fa4-4181-86e0-e754dd335ad3)

asad0206 commented:
@aadeshkulkarni can you please check the PR now ?

aadeshkulkarni commented:
@asad0206 - Please remove images (png, jpg, etc.) from the background as that adds performance problems due to longer loading time.

Use bg gradients or svgs

Some references:
https://dribbble.com/shots/22782075-Upload-Modal
https://www.epicweb.dev/tips/creating-glassmorphism-effects-with-tailwind-css
https://tailwindcss-glassmorphism.vercel.app/
